.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# automatically requests pull request reviews for files matching the given pattern; the last match takes precendence
+
+*       @spacetelescope/ssb-hstcal-maintainers
```

Comment:
Wait a min... none of the people in this group are still active maintainers of hstcal except for @mdlpstsci 

https://github.com/orgs/spacetelescope/teams/ssb-hstcal-maintainers

Currently I see:

* @jhunkeler (in ITSD now)
* @sosey (in INS now)
* myself (in DATB now)
* @stsci-hack (retired)
* @mdlpstsci (the active maintainer)

Reply:
ah good catch! I used the existing team without checking that. It looks like @jhunkeler is currently the team maintainer; Joe, when you have time (and get back from vacation) could you make @mdlpstsci a maintainer of the team? You could also remove yourself, @pllim, and @sosey if they don't want the notification pings for new PRs. 

It looks like @stsci-hack will be contracting until September so I assume he can stay